src/ocean/util/log/Config.d

```diff
@@ -315,7 +315,9 @@ public Layout newLayout ( istring layout_str )
             break;
 
         default:
-            throw new Exception("Invalid layout requested : " ~ layout_str);
+            istring msg = "Invalid layout requested : ";
```

Comment:
Ha gotcha. Pretty please split the file & line change into another commit? Just a minor thing, but I think that'll make the other change here -- which is actually connected to this commit -- easier to spot.